DeleteLLVMComments.java

```diff
@@ -0,0 +1,56 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+// Copyright (c) 2015 K Team. All Rights Reserved.
+public class DeleteLLVMComments {
+
+    /**
+     * @author LiyiLi
+     * This java code takes in a LLVM program and
+     * eliminates the LLVM comments in the program, then
+     * outputs the modified LLVM program
+     * to a temporary LLVM file.
+     */
+    public static void main(String[] args) {
+        String fileName = args[0];
+        File llvmFile = new File(fileName);
+        if(llvmFile.isFile()){
+            try {
+                Scanner scanner = new Scanner(llvmFile);
+                PrintWriter newLlvmProgam = new PrintWriter("llvmtemp.ll");
```

Comment:
I am trying to make a xml configuration and use ktest to test all programs now. Previously, I don't have the whole piece of LLVM syntax so that I found it was useless to have a ktest file to test all programs by LLVM syntax. If you don't have the whole LLVM syntax, then you will get basically errors for all input programs. Now, it is the time to set up the ktest file.